File: src/python/bgb/player/playermanager.py

class PlayerManager:
    players = []
    
    def add_player(self, player):
        if len(self.players) < 2:
            self.players.append(player)
        else:
            raise Exception("Too many players")
        
    def players_ready(self):
        if len(self.players) == 2:
            return True
        else:
            return False

File: src/java/examples/src/com/google/code/bgb/tictactoe/player/PlayerManager.java
package com.google.code.bgb.tictactoe.player;

import java.util.ArrayList;
import java.util.Collection;


public class PlayerManager {

	private Collection<Player> players;
	
	public PlayerManager() {
		players = new ArrayList<Player>();
	}
	
	public void addPlayer(Player player) {
		if(players.size() >= 2) {
			throw new IllegalStateException("Too many players");
		}
		
		players.add(player);
	}
	
	public boolean playersReady() {
		return (players.size() == 2);
	}
}
